runtime/src/compute_budget_details.rs

```diff
@@ -23,7 +23,14 @@ pub trait GetComputeBudgetDetails {
         instructions: impl Iterator<Item = (&'a Pubkey, &'a CompiledInstruction)>,
         _round_compute_unit_price_enabled: bool,
     ) -> Option<ComputeBudgetDetails> {
-        let compute_budget_limits = process_compute_budget_instructions(instructions).ok()?;
+        // ComputeBudgetDetails does not concern with loaded_accounts_data_size_limit, hence safe
+        // to hardcode its feature gate `default_loaded_accounts_data_size_limit` as activated
+        let use_default_loaded_accounts_data_size = true;
+        let compute_budget_limits = process_compute_budget_instructions(
```

Comment:
can we change this variable to `compute_budget_details`?

`ComputeBudgetLimits` **does** in fact deal with the loaded account size; it's `ComputeBudgetDetails` that doesn't.

(though this naming is confusing since I'd maybe intuitively think `Details` has more details?)

Reply:
Yea think `..._deails` might be more confusing. `process_compute_budget_instructions(...) -> Result< ComputeBudgetLimits, ...>`, `...._limits` reflects better that those are the limits we got from processing instructions.